tests/test_update.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
```

Comment:
The name `test_update` is too generic since we have `update_bounds`, `update_initital_conditions`, but also `update_objective_functions` and `update_constraints`. If you plan to move every updates in here, this is fine. If you just plan to put bounds and initial_conditions, maybe `test_update_bounds_and_init.py` would be a better name